fe/fe-core/src/main/java/com/starrocks/common/util/CompressionUtils.java

```diff
@@ -25,4 +25,22 @@ public class CompressionUtils {
     public static TCompressionType findTCompressionByName(String name) {
         return tCompressionByName.get(name);
     }
+
+    // Return TCompressionType according to input name for data page compression.
+    // Return null if input name is an invalid compression type for data page compression.
+    public static TCompressionType findPageTCompressionByName(String name) {
```

Comment:
getCompressTypeByName is OK. FE does not know the page concept.

Reply:
Yes, but the compression check must be done in fe as we can return the error in analyzeCompressionType immediately if we get a unsupported compression type. So I think it is necessary to do the check in fe?

Reply:
My meaning is the function name can be changed. `getCompressTypeByName` may be better